app/services/ValidateMetadataService.scala

```diff
@@ -0,0 +1,29 @@
+package services
+
+import configuration.GraphQLConfiguration
+import play.api.libs.ws.WSClient
+import play.api.{Configuration, Logging}
+
+import java.util.UUID
+import javax.inject.Inject
+import scala.concurrent.{ExecutionContext, Future}
+
+class ValidateMetadataService @Inject() (val ws: WSClient, val configuration: Configuration, graphQLConfiguration: GraphQLConfiguration)(implicit
+    val executionContext: ExecutionContext
+) extends Logging {
+
+  def triggerMetadataValidation(consignmentId: UUID, token: String): Future[Boolean] = {
+    val url = s"${configuration.get[String]("metadatavalidation.baseUrl")}/draft-metadata/validate/$consignmentId"
+    ws.url(url)
+      .addHttpHeaders(("Authorization", token), ("Content-Type", "application/json"))
+      .post("{}")
+      .flatMap(r =>
+        r.status match {
+          case 200 => Future(true)
```

Comment:
Can you use `Future.successful(true)` instead of `Future(true)` because it is much faster and reliable?

https://blog.colinbreck.com/future-successful-blocking-your-future-success/